_studio/hevce_hw/h265/include/plugin_version_linux.h

```diff
@@ -23,22 +23,22 @@
 #pragma once
 
 /* These string constants set Media SDK version information for Linux, Android, OSX. */
-#ifndef MFX_PLUGIN_FILE_VERSION
-#define MFX_PLUGIN_FILE_VERSION "0.0.0.0"
-#endif
 #ifndef MFX_PLUGIN_PRODUCT_NAME
 #define MFX_PLUGIN_PRODUCT_NAME "Intel(R) Media SDK Plugin"
 #endif
-#ifndef MFX_PLUGIN_PRODUCT_VERSION
-#define MFX_PLUGIN_PRODUCT_VERSION "0.0.000.0000"
-#endif
 #ifndef MFX_FILE_DESCRIPTION
 #define MFX_FILE_DESCRIPTION "File info ought to be here"
 #endif
 
-const char* g_MfxProductName = "mediasdk_product_name: " MFX_PLUGIN_PRODUCT_NAME;
-const char* g_MfxCopyright = "mediasdk_copyright: Copyright(c) 2007-2018 Intel Corporation";
-const char* g_MfxProductVersion = "mediasdk_product_version: " MFX_PLUGIN_PRODUCT_VERSION;
-const char* g_MfxFileVersion = "mediasdk_file_version: " MFX_PLUGIN_FILE_VERSION;
+#define STRINGIZE(s) #s
```

Comment:
Why do we need 2 defines when one of them call another just for only one purpose - converting marco to string ?
I mean why don't you squash 2 define to 1 :
#define CONVERT_TO_STRING(s) #s